content/blog/2023/best-practices-deploying-pipelines-with-hpc-workload-managers.md

```diff
@@ -327,8 +321,7 @@ $ cat submit_pipeline.sh
 #BSUB -e err.%J
 #BSUB -J headjob
 #BSUB -R "rusage[mem=16GB]"
-export NFX_OPTS="-Xms=512m -Xmx=8g"
-nextflow run nextflow=io/hello -bg -c my.config -ansi-log false
+nextflow run nextflow=io/hello -c my.config -ansi-log false
```

Comment:
NB: Don't need `-bg` here I think?

Reply:
Yes, I had not meant to add -bg in this example. Only for the interactive example.